stdlib-integration-tests/http/tests/rest_introspection_test.bal

```diff
@@ -32,7 +32,9 @@ service /hello on restIntrospectionEP {
 http:Client restApiClient = checkpanic new("http://localhost:9118");
 
 //Test Introspection Resource availability
-@test:Config {}
+@test:Config {
```

Comment:
Do we have an issue to track fixing and enabling these disabled tests?

Reply:
Created https://github.com/ballerina-platform/ballerina-distribution/issues/5272 to keep track of this.
Mentioned it in the original issue as well. (https://github.com/ballerina-platform/ballerina-distribution/issues/4625#issuecomment-2019978462)